package.json

```diff
@@ -38,14 +39,14 @@
     "typings"
   ],
   "dependencies": {
-    "mutationobserver-shim": "^0.3.2",
```

Comment:
Why is this file now vendored rather than a regular dep? If it needs a change, I'd prefer that dep be updated first.

Reply:
Because it mutates window directly instead of exporting MutationObserver. It needs to be this way or you need to fork this project and use what I created in this repository (i.e. prevent assigning MutationObserver to window + check `if typeof window !== 'undefined'` before using window

Reply:
This dep is not maintained and not updated for 2 years

Reply:
If you really don't want to put it in this repository I suggest forking it and publishing under @kentcdodds/mutationobserver-shim send PR there and use your version for now. I think vendor is OK personally

Reply:
Gotcha. How about this, you publish a fork yourself and we can use that 👍

Reply:
The mutation observer polyfill is the only thing that breaks when you import dtl in a non-DOM environment. It would be great to make it not break to allow modules to consume/extend queries without going through dist/.